Makefile

```diff
@@ -22,6 +23,12 @@ vendor/ramda.js: node_modules/ramda/dist/ramda.js
 vendor/%.js: node_modules/%/index.js
 	cp '$<' '$@'
 
+js/hm-search.js: node_modules/hindley-milner-search/hm-search.js Makefile
+	$(BROWSERIFY) --standalone HMS -- '$<' >'$@'
```

Comment:
browser standalone files included in v0.2.3. You should see `hms.js` and `hms.min.js` now. Exported as `HMS` as you did.